module/cmd/gravity/cmd/genaccounts.go

```diff
@@ -151,7 +151,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			bankGenState := banktypes.GetGenesisStateFromAppState(depCdc, appState)
 			bankGenState.Balances = append(bankGenState.Balances, balances)
 			bankGenState.Balances = banktypes.SanitizeGenesisBalances(bankGenState.Balances)
-
+			bankGenState.Supply = bankGenState.Supply.Add(balances.Coins...)
```

Comment:
Could you provide the motivation for setting this? Is supply not calculated automatically from the balances?

Reply:
I find this [unalignment](https://github.com/cosmos/cosmos-sdk/blob/3a7a2645f05f0aecdf96ebda14a7c62f6d4ddc55/x/auth/helpers/genaccounts.go#L122) when fix `panic: genesis supply is incorrect` when fix [sim](https://github.com/crypto-org-chain/gravity-bridge/pull/218) tests